ServiceApp/src/org/droidplanner/services/android/core/mission/survey/SurveyImpl.java

```diff
@@ -115,6 +119,26 @@ public static msg_mission_item packSurveyPoint(LatLong point, double altitude) {
         return mavMsg;
     }
 
+    protected msg_mission_item getYawCondition(double angle){
```

Comment:
@TShapinsky make the following two methods `private` unless you're planning to use/override them in another class.